specification/http/src/main/scripts/org/kaazing/specification/http/rfc7230/connection.management/client.with.pipelining.must.not.retry.pipelining.immediately.after.failure/request.rpt

```diff
@@ -29,6 +29,7 @@ read "HTTP/1.1 200 OK\r\n"
 read "Content-Length: 8\r\n"
 read "\r\n"
 read "request1"
+closed
```

Comment:
`read closed` can be used in a http script. The client script is a tcp script, so we cannot use it. Also, `read closed` does not guarantee that the connection is closed. It just signals to the driver that the request is ended.
From the k3po wiki:
> In order to inform the K3PO driver that no more headers are expected and that it is alright to flush the request to the wire, a read close or write close is expected after the request or response.
Nothing about closing the connection.